.lefthook.yml

```diff
@@ -0,0 +1,13 @@
+assert_lefthook_installed: true
```

Comment:
just run `pnpm i` to install it. This install step will setup the git pre-commit hook automatically